packages/runtime/src/helpers/edge.ts

```diff
@@ -214,7 +214,7 @@ export const writeEdgeFunctions = async (netlifyConfig: NetlifyConfig) => {
       path: '/_next/image*',
     })
   }
-  if (!process.env.NEXT_DISABLE_NETLIFY_EDGE) {
+  if (!(process.env.NEXT_DISABLE_NETLIFY_EDGE !== 'false' && process.env.NEXT_DISABLE_NETLIFY_EDGE !== '0')) {
```

Comment:
```suggestion
  if (process.env.NEXT_DISABLE_NETLIFY_EDGE !== ‘true` && process.env.NEXT_DISABLE_NETLIFY_EDGE !== ‘1’) {
```

Reply:
I believe using DeMorgan's law, these two expressions should use the logical OR. I'll make that change.

Reply:
Thanks! I'll make this change